### DISCUSSION TITLE
Srk mini e3 v3 drive

### DISCUSSION THREAD

**Discussion by caohuongls** [grblHAL/core#506] [Q&A]
Does GRBR-Hal compatible with srk mini e3 v3 and how to config it ?

**Comment by terjeio** [comment]
[Yes](https://github.com/grblHAL/Controllers), the firmware can be built with the [Web Builder](http://svn.io-engineering.com:8080/?driver=STM32F1xx&board=BTT%20SKR%20MINI%20E3%20V2.0%204-axis%20%28Bootloader%29).

Configuration is similar to legacy Grbl, via [$-settings](https://github.com/grblHAL/core/wiki/Additional-or-extended-settings).
